test/evergreen.yml

```diff
@@ -1583,10 +1592,11 @@ buildvariants:
   run_on:
   - ubuntu1804-test
   expansions:
-    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd)
+    test_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH LD_LIBRARY_PATH=$(pwd)/.libs top_srcdir=$(pwd)/.. top_builddir=$(pwd) CFLAGS="-fPIC -ggdb"
```

Comment:
Should the addition of CFLAGS setting be put as part of posix_configure_flags?
Same comment for other POSIX build variants below.

Reply:
Just following the convention from other tasks in the file. I believe all existing tasks/function add CFLAGS under `configure_env_vars`.

Reply:
Fixed the flag placement. Need to wait for RHEL and OSX job to confirm changes are working as expected.